examples/troopjs_require/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "private": true,
+  "dependencies": {
+    "jquery": "^2",
+    "when": "^3.4",
+    "poly": "git://github.com/cujojs/poly#0.6.0",
+    "requirejs": "^2.1",
+    "todomvc-app-css": "^2",
```

Comment:
@tastejs/todomvc This app already updated to classes, which is good, but we will need to wait to land the PR before the tests are updated right?